docs/howto/resource-estimation.rst

```diff
@@ -40,9 +40,16 @@ Maximum memory allowed per user
 
 Depending on what kinda work your users are doing, they will use different amounts
 of memory. The easiest way to determine this is to run through a typical user
-workflow yourself, and measure how much memory is used. Add 20-40% headroom for
-users to 'play around', and that should be the maximum memory allowed per user.
-The system will prevent users from using more memory than this.
+workflow yourself, and measure how much memory is used.
+
+For example, you can begin running a Jupyter Notebook session on your JupyterHub, then open a
```

Comment:
This is great for now! I wanna add nbresuse to the default and use that instead though.

Reply:
yeah I was gonna suggest the same thing, +1 from me on adding by default